crates/bevy_ecs/src/label.rs

```diff
@@ -1,10 +1,14 @@
 //! Traits used by label implementations
 
-use std::{
+use core::{
     any::Any,
     hash::{Hash, Hasher},
 };
 
+// Re-exported for use within `define_label!`
+#[doc(hidden)]
+pub use alloc::boxed::Box;
```

Comment:
This is required for the `define_label!` macro. It's definitely not ideal, and I'd like to move it to a dedicated `bevy_alloc` crate instead, but it's a minimal issue IMO.